data.sql

```diff
@@ -0,0 +1,21 @@
+\d animals;
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '2020-2-3', 0, true, 10.23 );
+SELECT * FROM animals;
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8 ); 
+SELECT * FROM animals;
+UPDATE animals SET neutered = false WHERE id = 1;
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021-1-7' , 1, false , 15.04 ), (4, 'Devimon', '2017-5-12', 5, true,11);
+SELECT * FROM animals;
+
+-- The data for the second day's project
+
+ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+ VALUES (5, 'Charmander', '2020-2-8', 0, 'false', 11),
```

Comment:
- [ ] It would be better if we input the required weight as we have in the project requirements
![Screenshot from 2023-05-11 14-34-18](https://github.com/IsmailChangezi/vet-clinic/assets/46388113/ccedc369-08d8-48de-a92f-7cee8c6eada2)
Let us kindly change to `-11` instead